### ISSUE TITLE
Missing aria-modal=true attribute

### ISSUE THREAD

**Issue by gnoyixiang** [react-component/dialog#184]
Hello, based on the W3 documentation on dialogs, it is stated that

>  The dialog container element has aria-modal set to true.

I see this in other react accessibility based libraries such as ReachUI and Reakit. Am hoping that the attribute `[aria-modal=true]` can be added in.

https://www.w3.org/TR/wai-aria-practices-1.2/#dialog_roles_states_props

**Comment by afc163** [comment]
https://github.com/react-component/dialog/blob/4f70824474b6988e79b8461341b37dce23ea19ff/src/Dialog/Content/Panel.tsx#L114
